ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -222,6 +247,8 @@ A single compaction job could be performed across multiple compaction cycles (if
     logger.trace("resumeCompaction() started for {}", storeId);
     runningLatch = new CountDownLatch(1);
     compactionInProgress.set(true);
+    selectedContainers.clear();
+    getAllContainersByStatus(EnumSet.of(Container.ContainerStatus.INACTIVE, Container.ContainerStatus.DELETE_IN_PROGRESS));
```

Comment:
Can you log the selectedContainers as info log before running into compaction?

Reply:
Updated